src/core/Track.cpp

```diff
@@ -708,7 +708,6 @@ void TrackContentObjectView::paintTextLabel(QString const & text, QPainter & pai
 		elidedPatternName = text.trimmed();
 	}
 
-	painter.fillRect(QRect(0, 0, width(), fontMetrics.height() + 2 * textTop), textBackgroundColor());
```

Comment:
I disagree with this change. We need this to ensure text stays legible, especially since we allow clips to be colored. I think you should either let the rectangle display on top of the fade curve, or adjust the fade curve's height so that it doesn't share any space with the rectangle.

Reply:
I changed it now. With the text, it looks a bit strange when not zoomed in. Maybe the fade handles should only be displayed when zoomed in close enough?

Reply:
I think it looks great now while zoomed in! I'll have to test it out before I can give an opinion on different zoom levels, I'll try to do that tomorrow.

Reply:
I think it looks fine zoomed out, it's easy enough to see the duration of the fades. That being said, the boxes representing the handles seem to be drawn too low, with a regular height clip you have to press above the box in order to adjust the fade.